pupil-api/sshd_config

```diff
@@ -0,0 +1,16 @@
+# This is ssh server systemwide configuration file.
+#
+# /etc/sshd_config
+
```

Comment:
Again, a note to say that this config should not be exposed publicly would welcome.